### DISCUSSION TITLE
About codeactions

### DISCUSSION THREAD

**Discussion by yigitkalan** [neoclide/coc.nvim#4588] [General]
I want to get suggestions or quickfixes for a problem in my code with coc, i use the below line in my config:
```
keyset("n", "<leader>ef", "<Plug>(coc-fix-current)", opts)
```
as in the example config but i get : 
```
E15: Invalid expression <80><fd>S(coc-fix-current)
```
What am i missing ?

**Comment by fannheyward** [comment]
Have no idea about this, I don't use Lua to config mappings, do you have `replace_keycodes` in your keyset opts?

**Reply by yigitkalan** [reply]
I have mine like this :
```
local opts = {silent = true, noremap = true, expr = true, replace_keycodes = false}
```
The issue persist when i enabled the replace_keycodes.

**Comment by yigitkalan** [comment]
Okay i think i found a solution i take the whole example vim script config and put it in :
```
vim.cmd[[
    ...
 ]]
```
This error was probably was beause of the config, the lua example might be wrong.
